lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -206,6 +206,7 @@ function dosomething_reportback_form($form, &$form_state, $entity = NULL) {
   }
 
   $form['reportback_inputs']['caption'] = array(
+    '#prefix' => '<h2>' . t('Your impact') . '</h2>',
```

Comment:
Make this an `h4` but add these classes to make it look like an h2: `heading -beta`